itests/shellenv.feature

```diff
@@ -0,0 +1,13 @@
+Feature: env variables
+
+Scenario: JBANG_RUNTIME_SHELL available
```

Comment:
@quintesse  what is the use of this variable?

Reply:
The runtime scripts tell JBang what shell (eg. *nix sh/bash/etc or CMD or Powershell) it was started from. There's just no good way to otherwise detect what kind of command line we should be generating.